docs/photoniq/fingerprinting/get-started-fingerprinting.md

```diff
@@ -0,0 +1,268 @@
+---
+sidebar_position: 10
+title: Get Started with Fingerprinting
+---
+
+To incorporate the Digital Fingerprinting client into your website, add one of the following example scripts to your web page.
+
+## Fingerprint on Button Click
+
+In this example, the JavaScript is initially loaded at page load time, and a function is created that is tied to a button. Once the button is clicked, a call is made to the fingerprint server, which returns the visitorId. The visitorId is returned in the response header (x-photoniq-vid) and the JSON body’s response (visitorId). Remember to change  DS_URL to the provided values.
```

Comment:
The _Remember to change DS_URL to the provided values._ is a bit awkward.

Reply:
What are the "provided values" that they should change the DS_URL placeholder to?